config/samples/functions/python.yaml

```
@@ -19,7 +19,6 @@ metadata:
 spec:
   runtime: python
   responseIsEvent: true
```

well, I not exactly what I meant 😅
let's have one sample function public 

```suggestion
  responseIsEvent: true
  adapterOverrides:
    public: true
```